testnet/tools/nns-tools/lib/proposals.sh

```diff
@@ -83,40 +87,83 @@ generate_nns_upgrade_proposal_text() {
     fi
 
     OUTPUT=$(
-        cat <<EOF
-## Proposal to Upgrade the $CAPITALIZED_CANISTER_NAME Canister
-### Proposer: DFINITY Foundation
-### Git Hash: $NEXT_COMMIT
-### New Wasm Hash: $WASM_SHA$([ ! -z "$ARGS_HASH" ] && echo $'\n'"### Upgrade Args Hash: $ARGS_HASH")
-### Target canister: $(nns_canister_id "$CANISTER_NAME")
----
-## Features
-TODO ADD FEATURE NOTES
-## Release Notes
+        cat <<++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+# Upgrade the $CAPITALIZED_CANISTER_NAME Canister to Commit $SHORT_NEXT_COMMIT
+
+__Proposer__: ${PROPOSER}\\
+__Source Code__: [$NEXT_COMMIT][new-commit]
```

Comment:
I'm not sure, but I don't think this is the standard link format - it's usually [TEXT](URL)